launcher/BaseInstance.cpp

```diff
@@ -69,6 +69,7 @@ BaseInstance::BaseInstance(SettingsObjectPtr globalSettings, SettingsObjectPtr s
     m_settings->registerSetting("lastTimePlayed", 0);
 
     m_settings->registerSetting("linkedInstances", "[]");
+    m_settings->registerSetting("shortcuts", QVariant::fromValue(QList<ShortcutData>{}));
```

Comment:
Iwould suggest to json encode any settings that is more complex than basic types

Reply:
Sure